tools/gltfViewer/README.md

```diff
@@ -0,0 +1,39 @@
+# Simple glTL Viewer
+
+The gltfViewer application can be use to view glTF files, including
+selecting cameras and playing animations.
+
+Usage: `gltf [-f] [-c name] [-p name] [-l] file.gltf`
+
+## Options
+
+*  `-h, --help`               : Displays help.
+*  `-v, --version`            : Displays version information.
+*  `-f, --fullScreen`         : show full screen.
+*  `-p, --play <animations>`  : play named animation
```

Comment:
plays, loops. Or instead of Displays, display. 
Just be sure to use capital case or not for all of them, and third person or not for all of them

Reply:
done